cookiecutter/generate.py

```diff
@@ -284,8 +287,15 @@ def generate_files(repo_dir, context=None, output_dir='.',
 
     _run_hook_from_repo_dir(repo_dir, 'pre_gen_project', project_dir, context)
 
+    # Configure jinja templates searchpath
+    templates = ['.']
+    if extra_templates is not None:
+        if isinstance(extra_templates, basestring):
```

Comment:
The doc string for this parameter reads that it is expected to be a list, so this instance check is always going to fail. Is it the items that you want to check?